core/src/main/scala/ai/h2o/sparkling/frame/H2OFrame.scala

```diff
@@ -35,9 +34,11 @@ import water.api.schemas3._
 /**
  * H2OFrame representation via Rest API
  */
-class H2OFrame private(val frameId: String, val columns: Array[H2OColumn], val chunks: Array[H2OChunk]) extends Serializable {
+class H2OFrame private(val frameId: String, val columns: Array[H2OColumn], val chunks: Array[H2OChunk])
+  extends Serializable
+    with RestEncodingUtils {
```

Comment:
nit: indentation, maybe I would merge this and the previous line

Reply:
👍